[2024-10-10T11:43:26.071+0700] {processor.py:157} INFO - Started process (PID=2768) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:43:26.072+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:43:26.074+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:43:26.074+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:43:26.113+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:43:26.110+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:43:26.114+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:43:26.113+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:43:26.115+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:43:26.143+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.079 seconds
[2024-10-10T11:48:37.424+0700] {processor.py:157} INFO - Started process (PID=2908) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:48:37.469+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:48:37.471+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:48:37.471+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:48:37.680+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:48:37.677+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:48:37.681+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:48:37.681+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:48:37.682+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:48:37.703+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.286 seconds
[2024-10-10T11:51:24.536+0700] {processor.py:157} INFO - Started process (PID=3095) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:51:24.599+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:51:24.601+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:51:24.601+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:51:24.656+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:51:24.655+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:51:24.657+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:51:24.657+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:51:24.657+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:51:24.685+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.169 seconds
[2024-10-10T11:53:23.577+0700] {processor.py:157} INFO - Started process (PID=3343) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:53:23.589+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:53:23.592+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:53:23.591+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:53:23.600+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:53:23.599+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:53:23.601+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:53:23.601+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:53:23.601+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:53:23.630+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2024-10-10T11:54:21.269+0700] {processor.py:157} INFO - Started process (PID=3403) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:54:21.273+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:54:21.275+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:54:21.275+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:54:21.282+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:54:21.280+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:54:21.282+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:54:21.282+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:54:21.283+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:54:21.303+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-10-10T11:55:25.892+0700] {processor.py:157} INFO - Started process (PID=3483) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:55:25.915+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:55:25.918+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:55:25.917+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:55:25.926+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:55:25.924+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:55:25.926+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:55:25.926+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:55:25.927+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:55:25.948+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2024-10-10T11:56:26.423+0700] {processor.py:157} INFO - Started process (PID=3549) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:56:26.434+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:56:26.436+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:56:26.435+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:56:26.443+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:56:26.441+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:56:26.444+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:56:26.444+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:56:26.445+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:56:26.465+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2024-10-10T11:57:24.440+0700] {processor.py:157} INFO - Started process (PID=3614) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:57:24.461+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:57:24.464+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:57:24.463+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:57:24.472+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:57:24.471+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:57:24.473+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:57:24.473+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:57:24.474+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:57:24.494+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2024-10-10T11:58:24.626+0700] {processor.py:157} INFO - Started process (PID=3686) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:58:24.647+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:58:24.650+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:58:24.649+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:58:24.658+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:58:24.656+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:58:24.658+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:58:24.658+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:58:24.659+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:58:24.680+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2024-10-10T11:59:35.492+0700] {processor.py:157} INFO - Started process (PID=3763) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:59:35.502+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T11:59:35.504+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:59:35.504+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:59:35.537+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:59:35.510+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T11:59:35.538+0700] {logging_mixin.py:149} INFO - [2024-10-10T11:59:35.538+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T11:59:35.539+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T11:59:35.563+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2024-10-10T12:00:32.820+0700] {processor.py:157} INFO - Started process (PID=3820) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:00:32.842+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:00:32.845+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:00:32.844+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:00:32.852+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:00:32.850+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:00:32.853+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:00:32.853+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:00:32.853+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:00:32.874+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2024-10-10T12:01:28.266+0700] {processor.py:157} INFO - Started process (PID=3880) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:01:28.268+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:01:28.270+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:01:28.269+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:01:28.275+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:01:28.274+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:01:28.276+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:01:28.276+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:01:28.276+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:01:28.296+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-10-10T12:02:18.521+0700] {processor.py:157} INFO - Started process (PID=3939) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:02:18.533+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:02:18.536+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:02:18.535+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:02:18.544+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:02:18.543+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:02:18.545+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:02:18.544+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:02:18.545+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:02:18.565+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2024-10-10T12:03:08.727+0700] {processor.py:157} INFO - Started process (PID=3992) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:03:08.768+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:03:08.771+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:03:08.770+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:03:08.778+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:03:08.776+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:03:08.779+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:03:08.778+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:03:08.779+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:03:08.800+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.080 seconds
[2024-10-10T12:04:01.208+0700] {processor.py:157} INFO - Started process (PID=4049) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:04:01.234+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:04:01.237+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:04:01.237+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:04:01.245+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:04:01.244+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:04:01.246+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:04:01.246+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:04:01.246+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:04:01.267+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-10-10T12:04:53.535+0700] {processor.py:157} INFO - Started process (PID=4162) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:04:53.548+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:04:53.550+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:04:53.550+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:04:53.558+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:04:53.556+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:04:53.558+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:04:53.558+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:04:53.559+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:04:53.580+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.052 seconds
[2024-10-10T12:05:47.580+0700] {processor.py:157} INFO - Started process (PID=4223) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:05:47.604+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:05:47.607+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:05:47.606+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:05:47.616+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:05:47.614+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:05:47.617+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:05:47.616+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:05:47.617+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:05:47.638+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-10-10T12:06:44.205+0700] {processor.py:157} INFO - Started process (PID=4284) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:06:44.222+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:06:44.225+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:06:44.224+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:06:44.233+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:06:44.231+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:06:44.233+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:06:44.233+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:06:44.234+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:06:44.254+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2024-10-10T12:07:40.221+0700] {processor.py:157} INFO - Started process (PID=4340) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:07:40.234+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:07:40.237+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:07:40.236+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:07:40.245+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:07:40.243+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:07:40.245+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:07:40.245+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:07:40.246+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:07:40.266+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2024-10-10T12:08:38.561+0700] {processor.py:157} INFO - Started process (PID=4408) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:08:38.563+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:08:38.564+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:08:38.564+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:08:38.570+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:08:38.568+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:08:38.570+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:08:38.570+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:08:38.571+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:08:38.591+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-10-10T12:09:32.775+0700] {processor.py:157} INFO - Started process (PID=4466) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:09:32.777+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:09:32.778+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:09:32.778+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:09:32.783+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:09:32.782+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:09:32.784+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:09:32.784+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:09:32.784+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:09:32.804+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-10-10T12:10:24.628+0700] {processor.py:157} INFO - Started process (PID=4519) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:10:24.630+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:10:24.632+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:10:24.631+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:10:24.637+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:10:24.636+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:10:24.637+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:10:24.637+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:10:24.638+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:10:24.658+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2024-10-10T12:11:17.024+0700] {processor.py:157} INFO - Started process (PID=4585) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:11:17.037+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:11:17.039+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:11:17.039+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:11:17.046+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:11:17.044+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:11:17.047+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:11:17.046+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:11:17.047+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:11:17.069+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2024-10-10T12:12:10.362+0700] {processor.py:157} INFO - Started process (PID=4649) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:12:10.364+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:12:10.365+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:12:10.365+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:12:10.371+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:12:10.369+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:12:10.372+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:12:10.371+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:12:10.372+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:12:10.392+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-10-10T12:13:04.895+0700] {processor.py:157} INFO - Started process (PID=4710) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:13:04.913+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:13:04.916+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:13:04.915+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:13:04.924+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:13:04.923+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:13:04.925+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:13:04.925+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:13:04.925+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:13:04.945+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
[2024-10-10T12:13:55.468+0700] {processor.py:157} INFO - Started process (PID=4771) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:13:55.493+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:13:55.496+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:13:55.495+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:13:55.504+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:13:55.503+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:13:55.505+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:13:55.505+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:13:55.505+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:13:55.525+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2024-10-10T12:15:09.920+0700] {processor.py:157} INFO - Started process (PID=4857) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:15:09.950+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:15:09.953+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:15:09.952+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:15:09.961+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:15:09.959+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:15:09.962+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:15:09.961+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:15:09.962+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:15:09.983+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-10-10T12:16:01.533+0700] {processor.py:157} INFO - Started process (PID=4918) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:16:01.575+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:16:01.577+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:16:01.576+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:16:01.582+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:16:01.581+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:16:01.583+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:16:01.583+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:16:01.583+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:16:01.603+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2024-10-10T12:16:57.743+0700] {processor.py:157} INFO - Started process (PID=4984) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:16:57.745+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:16:57.747+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:16:57.747+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:16:57.752+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:16:57.751+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:16:57.753+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:16:57.753+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:16:57.753+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:16:57.772+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-10-10T12:17:51.924+0700] {processor.py:157} INFO - Started process (PID=5050) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:17:51.955+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:17:51.958+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:17:51.957+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:17:51.967+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:17:51.965+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:17:51.968+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:17:51.967+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:17:51.968+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:17:51.987+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-10-10T12:18:45.510+0700] {processor.py:157} INFO - Started process (PID=5112) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:18:45.511+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:18:45.513+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:18:45.513+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:18:45.518+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:18:45.517+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:18:45.519+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:18:45.519+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:18:45.519+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:18:45.538+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-10-10T12:19:38.479+0700] {processor.py:157} INFO - Started process (PID=5168) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:19:38.511+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:19:38.514+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:19:38.513+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:19:38.523+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:19:38.522+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:19:38.524+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:19:38.524+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:19:38.524+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:19:38.543+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2024-10-10T12:20:30.533+0700] {processor.py:157} INFO - Started process (PID=5225) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:20:30.535+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:20:30.537+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:20:30.536+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:20:30.542+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:20:30.541+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:20:30.543+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:20:30.543+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:20:30.543+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:20:30.562+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-10-10T12:21:29.599+0700] {processor.py:157} INFO - Started process (PID=5306) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:21:29.652+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T12:21:29.654+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:21:29.653+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:21:29.661+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:21:29.659+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T12:21:29.662+0700] {logging_mixin.py:149} INFO - [2024-10-10T12:21:29.662+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T12:21:29.663+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T12:21:29.693+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.100 seconds
[2024-10-10T13:20:44.988+0700] {processor.py:157} INFO - Started process (PID=2001) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T13:20:45.017+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T13:20:45.018+0700] {logging_mixin.py:149} INFO - [2024-10-10T13:20:45.018+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T13:20:45.102+0700] {logging_mixin.py:149} INFO - [2024-10-10T13:20:45.074+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T13:20:45.103+0700] {logging_mixin.py:149} INFO - [2024-10-10T13:20:45.103+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T13:20:45.104+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T13:20:45.128+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.150 seconds
[2024-10-10T13:26:11.501+0700] {processor.py:157} INFO - Started process (PID=2642) to work on /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T13:26:11.751+0700] {processor.py:826} INFO - Processing file /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-10T13:26:11.753+0700] {logging_mixin.py:149} INFO - [2024-10-10T13:26:11.752+0700] {dagbag.py:541} INFO - Filling up the DagBag from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T13:26:12.002+0700] {logging_mixin.py:149} INFO - [2024-10-10T13:26:11.815+0700] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-10T13:26:12.003+0700] {logging_mixin.py:149} INFO - [2024-10-10T13:26:12.003+0700] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-10T13:26:12.004+0700] {processor.py:838} WARNING - No viable dags retrieved from /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-10T13:26:12.101+0700] {processor.py:179} INFO - Processing /home/hasyidan/venv/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.612 seconds
